### ISSUE TITLE
[4.0] Debug permissions per component

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#25401]
### Reproduce
From the users menu select either users or groups
Then select the permissions for any user/group
You should now have a table of the permissions
From the filter options button chose any component

### Expected 
Table of the permissions for that component

### Actual
`0 Cannot use object of type stdClass as array`

Call stack
--
```
# | Function | Location
1 | () | JROOT\administrator\components\com_users\Helper\UsersHelperDebug.php:91
2 | Joomla\Component\Users\Administrator\Helper\UsersHelperDebug::getDebugActions() | JROOT\administrator\components\com_users\Model\DebuguserModel.php:65
3 | Joomla\Component\Users\Administrator\Model\DebuguserModel->getDebugActions() | JROOT\libraries\src\MVC\View\AbstractView.php:146
4 | Joomla\CMS\MVC\View\AbstractView->get() | JROOT\administrator\components\com_users\View\Debuguser\HtmlView.php:98
5 | Joomla\Component\Users\Administrator\View\Debuguser\HtmlView->display() | JROOT\libraries\src\MVC\Controller\BaseController.php:699
6 | Joomla\CMS\MVC\Controller\BaseController->display() | JROOT\administrator\components\com_users\Controller\DisplayController.php:120
7 | Joomla\Component\Users\Administrator\Controller\DisplayController->display() | JROOT\libraries\src\MVC\Controller\BaseController.php:737
8 | Joomla\CMS\MVC\Controller\BaseController->execute() | JROOT\libraries\src\Dispatcher\ComponentDispatcher.php:146
9 | Joomla\CMS\Dispatcher\ComponentDispatcher->dispatch() | JROOT\libraries\src\Component\ComponentHelper.php:383
10 | Joomla\CMS\Component\ComponentHelper::renderComponent() | JROOT\libraries\src\Application\AdministratorApplication.php:116
11 | Joomla\CMS\Application\AdministratorApplication->dispatch() | JROOT\libraries\src\Application\AdministratorApplication.php:159
12 | Joomla\CMS\Application\AdministratorApplication->doExecute() | JROOT\libraries\src\Application\CMSApplication.php:241
13 | Joomla\CMS\Application\CMSApplication->execute() | JROOT\administrator\includes\app.php:63
14 | require_once() | JROOT\administrator\index.php:36

```

I have traced that error back to #22834 by @chmst 
Sorry but I don't know the solution espec as I cant replicate the bug that #22834 was fixing

**Comment by ghost** [comment]
Closed as having Pull Request #25402

**Comment by richard67** [comment]
@franz-wohlkoenig Not really closed up to now.
